packages/zowe-explorer-ftp-extension/CHANGELOG.md

```diff
@@ -6,6 +6,8 @@ All notable changes to the "zowe-explorer-ftp-extension" extension will be docum
 
 ### Bug fixes
 
+- Renamed extension to `IBM z/OS FTP Plug-in for Zowe`. [#2990](https://github.com/zowe/zowe-explorer-vscode/issues/2990)
```

Comment:
Should this say `IBM z/OS FTP for Zowe Explorer` to match the new display name? 😋